app/templates/show-project.hbs

```diff
@@ -111,7 +111,7 @@
             <h2>Milestones</h2>
             <ul class="no-bullet milestones-list">
               {{#if model.milestones}}
-                {{#each model.milestones as | milestone |}}
+                {{#each revisedmilestones as | milestone |}}
```

Comment:
Is `revisedmilestones` a good name for this? They're not all revised.

Reply:
I am futureproofing, in preparation for another request to change subsequent requests for action X

Reply:
Or, the array of milestones is revised, even if each milestone is not.

Reply:
It could just be `milestones`. No reason to qualify them.

Reply:
I wanted to make sure it was different from model.milestones. ` revisedmilestones` reads better in the handlebars, as it's right after a check for `model.milestones`

Reply:
Agree - this should be computed or something. This should also be wrapped in a component if milestones needs to be munged for presentation - I'm afraid to see the skeletons in the `show-project` controller closet!

Reply:
It is computed, but it has to happen at the projects level as we already have a component for each milestone.  

We could make a `milestones` component, but that seems like overkill for just one computed, so it may as well stay in `show-project`

Reply:
If the markup is using ~`revised`~`milestones` shouldn't the check be for it, not `model.milestones`?

Reply:
Meh. Semantics. This works. I'm merging it… 